COPYRIGHT.sh

```diff
@@ -0,0 +1,13 @@
+# !/usr/bin/env bash
```

Comment:
Cool. This tool is very handy and useful. I have 2 suggestions:
1. We should probably rename this script, to make it more explicit.
2. What about providing an option to let people pass the files as params?

Reply:
for 2, would this format be good?

```
./gen_header.sh -f file1.py -f file2.py
```

This passes file1.py and file2.py to generate header

Reply:
Yes, looks good to me.